[
  {
    "message": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "level": "FATAL",
    "type": "PLAYGROUND",
    "test": "ast-sqlite-boot",
    "plugin": "cf7-better-ui",
    "details": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "log": "/tmp/tmp.Rh7jzMd7XM"
  },
  {
    "message": "Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121",
    "level": "DEPRECATED",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:23 UTC] PHP Deprecated:  Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/calicotek-ebay-dashboard-tools/index.php on line 54",
    "level": "WARNING",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:23 UTC] PHP Warning:  Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/calicotek-ebay-dashboard-tools/index.php on line 54",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121",
    "level": "DEPRECATED",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:24 UTC] PHP Deprecated:  Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121",
    "level": "DEPRECATED",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:35 UTC] PHP Deprecated:  Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/calicotek-ebay-dashboard-tools/index.php on line 54",
    "level": "WARNING",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:35 UTC] PHP Warning:  Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/calicotek-ebay-dashboard-tools/index.php on line 54",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121",
    "level": "DEPRECATED",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:36 UTC] PHP Deprecated:  Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121",
    "level": "DEPRECATED",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:45 UTC] PHP Deprecated:  Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/calicotek-ebay-dashboard-tools/index.php on line 54",
    "level": "WARNING",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:45 UTC] PHP Warning:  Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/calicotek-ebay-dashboard-tools/index.php on line 54",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121",
    "level": "DEPRECATED",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:46 UTC] PHP Deprecated:  Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Undefined variable $activation_email_address in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2605",
    "level": "WARNING",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:55 UTC] PHP Warning:  Undefined variable $activation_email_address in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2605",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Undefined variable $activation_key in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2610",
    "level": "WARNING",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:55 UTC] PHP Warning:  Undefined variable $activation_key in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2610",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Undefined variable $activation_email_address in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2605",
    "level": "WARNING",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:55 UTC] PHP Warning:  Undefined variable $activation_email_address in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2605",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Undefined variable $activation_key in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2610",
    "level": "WARNING",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:55 UTC] PHP Warning:  Undefined variable $activation_key in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2610",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Undefined variable $activation_email_address in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2605",
    "level": "WARNING",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:03 UTC] PHP Warning:  Undefined variable $activation_email_address in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2605",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Undefined variable $activation_key in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2610",
    "level": "WARNING",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:03 UTC] PHP Warning:  Undefined variable $activation_key in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2610",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Undefined variable $activation_email_address in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2605",
    "level": "WARNING",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:04 UTC] PHP Warning:  Undefined variable $activation_email_address in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2605",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Undefined variable $activation_key in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2610",
    "level": "WARNING",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:04 UTC] PHP Warning:  Undefined variable $activation_key in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2610",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Undefined variable $activation_email_address in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2605",
    "level": "WARNING",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:11 UTC] PHP Warning:  Undefined variable $activation_email_address in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2605",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Undefined variable $activation_key in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2610",
    "level": "WARNING",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:11 UTC] PHP Warning:  Undefined variable $activation_key in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2610",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Undefined variable $activation_email_address in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2605",
    "level": "WARNING",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:12 UTC] PHP Warning:  Undefined variable $activation_email_address in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2605",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Undefined variable $activation_key in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2610",
    "level": "WARNING",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:12 UTC] PHP Warning:  Undefined variable $activation_key in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2610",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Uncaught Error: Call to a member function getName() on null in /wordpress/wp-content/plugins/catalogo/modules/Catalog.php:488",
    "level": "FATAL",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:47 UTC] PHP Fatal error:  Uncaught Error: Call to a member function getName() on null in /wordpress/wp-content/plugins/catalogo/modules/Catalog.php:488\nStack trace:\n#0 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(1171): CatalogoPluginColumnOperations->getIdFromName(Array, 'ref')\n#1 /wordpress/wp-content/plugins/catalogo/catalogo.php(494): CatalogoPluginCatalogData->getTypeColumns(Array, Array)\n#2 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#6 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#7 {main}\n  thrown in /wordpress/wp-content/plugins/catalogo/modules/Catalog.php on line 488",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: _mysql_data_types_cache.mysql_type.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:55 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>CREATE TABLE wp_catalogoplugin_column(\n                id int(11) NOT NULL AUTO_INCREMENT ,\n                name char(32) NOT NULL ,\n                description char(128) ,\n                id_column_definition int NOT NULL,\n                PRIMARY KEY ( `id`  ),\n                FOREIGN KEY (`id_column_definition`) REFERENCES wp_catalogoplugin_column_definition(`id`)\n                ) </p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_catalogoplugin_column, id, int(11)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_catalogoplugin_column, name, char(32)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_catalogoplugin_column, description, char(128)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_catalogoplugin_column, id_column_definition, int</li>\n<li>Executing: CREATE TABLE &quot;wp_catalogoplugin_column&quot; (\n&quot;id&quot; integer PRIMARY KEY AUTOINCREMENT NOT NULL ON CONFLICT REPLACE DEFAULT 0,\n&quot;name&quot; text NOT NULL ON CONFLICT REPLACE DEFAULT '' COLLATE NOCASE,\n&quot;description&quot; text COLLATE NOCASE,\n&quot;id_column_definition&quot; integer NOT NULL ON CONFLICT REPLACE DEFAULT 0) | (no parameters)</li>\n<li>Executing: CREATE  INDEX  &quot;wp_catalogoplugin_column__&quot; ON &quot;wp_catalogoplugin_column&quot; (&quot;&quot;) | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_catalogoplugin_column, wp_catalogoplugin_column__, </li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: _mysql_data_types_cache.mysql_type.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE wp...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(474): dbDelta(Array)\n#3 /wordpress/wp-content/plugins/catalogo/catalogo.php(477): CatalogoPluginColumnOperations->createTable()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#8 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#9 {main}</pre>\n for query CREATE TABLE wp_catalogoplugin_column(\n                id int(11) NOT NULL AUTO_INCREMENT ,\n                name char(32) NOT NULL ,\n                description char(128) ,\n                id_column_definition int NOT NULL,\n                PRIMARY KEY ( `id`  ),\n                FOREIGN KEY (`id_column_definition`) REFERENCES wp_catalogoplugin_column_definition(`id`)\n                )  made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginColumnOperations->createTable, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:55 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>insert into wp_catalogoplugin_column(name, description, id_column_definition) values('ref', 'reference[4] NOT NULL',1)</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: insert into wp_catalogoplugin_column(name, description, id_column_definition) values(:param0 , :param1 ,1) | parameters: ref, reference[4] NOT NULL</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(399): WP_SQLite_DB->query('insert into wp_...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(403): CatalogoPluginColumnOperations->insert(Object(CatalogoPluginColumn))\n#3 /wordpress/wp-content/plugins/catalogo/catalogo.php(481): CatalogoPluginColumnOperations->insertGet(Object(CatalogoPluginColumn))\n#4 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#8 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#9 {main}</pre>\n for query insert into wp_catalogoplugin_column(name, description, id_column_definition) values('ref', 'reference[4] NOT NULL',1) made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginColumnOperations->insertGet, CatalogoPluginColumnOperations->insert, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:55 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select * from wp_catalogoplugin_column where name='ref'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select * from wp_catalogoplugin_column where name=:param0  | parameters: ref</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select * from w...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(448): wpdb->get_results('select * from w...')\n#3 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(404): CatalogoPluginColumnOperations->getByName('ref')\n#4 /wordpress/wp-content/plugins/catalogo/catalogo.php(481): CatalogoPluginColumnOperations->insertGet(Object(CatalogoPluginColumn))\n#5 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#8 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#9 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#10 {main}</pre>\n for query select * from wp_catalogoplugin_column where name='ref' made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginColumnOperations->insertGet, CatalogoPluginColumnOperations->getByName, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:55 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>insert into wp_catalogoplugin_column(name, description, id_column_definition) values('label', 'label[50]',2)</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: insert into wp_catalogoplugin_column(name, description, id_column_definition) values(:param0 , :param1 ,2) | parameters: label, label[50]</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(399): WP_SQLite_DB->query('insert into wp_...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(403): CatalogoPluginColumnOperations->insert(Object(CatalogoPluginColumn))\n#3 /wordpress/wp-content/plugins/catalogo/catalogo.php(481): CatalogoPluginColumnOperations->insertGet(Object(CatalogoPluginColumn))\n#4 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#8 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#9 {main}</pre>\n for query insert into wp_catalogoplugin_column(name, description, id_column_definition) values('label', 'label[50]',2) made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginColumnOperations->insertGet, CatalogoPluginColumnOperations->insert, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:55 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select * from wp_catalogoplugin_column where name='label'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select * from wp_catalogoplugin_column where name=:param0  | parameters: label</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select * from w...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(448): wpdb->get_results('select * from w...')\n#3 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(404): CatalogoPluginColumnOperations->getByName('label')\n#4 /wordpress/wp-content/plugins/catalogo/catalogo.php(481): CatalogoPluginColumnOperations->insertGet(Object(CatalogoPluginColumn))\n#5 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#8 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#9 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#10 {main}</pre>\n for query select * from wp_catalogoplugin_column where name='label' made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginColumnOperations->insertGet, CatalogoPluginColumnOperations->getByName, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "level": "FATAL",
    "type": "PLAYGROUND",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "log": "/tmp/tmp.FQlaYH0CjI"
  },
  {
    "message": "Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121",
    "level": "DEPRECATED",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:23 UTC] PHP Deprecated:  Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/calicotek-ebay-dashboard-tools/index.php on line 54",
    "level": "WARNING",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:23 UTC] PHP Warning:  Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/calicotek-ebay-dashboard-tools/index.php on line 54",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121",
    "level": "DEPRECATED",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:24 UTC] PHP Deprecated:  Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121",
    "level": "DEPRECATED",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:35 UTC] PHP Deprecated:  Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/calicotek-ebay-dashboard-tools/index.php on line 54",
    "level": "WARNING",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:35 UTC] PHP Warning:  Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/calicotek-ebay-dashboard-tools/index.php on line 54",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121",
    "level": "DEPRECATED",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:36 UTC] PHP Deprecated:  Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121",
    "level": "DEPRECATED",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:45 UTC] PHP Deprecated:  Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/calicotek-ebay-dashboard-tools/index.php on line 54",
    "level": "WARNING",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:45 UTC] PHP Warning:  Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/calicotek-ebay-dashboard-tools/index.php on line 54",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121",
    "level": "DEPRECATED",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:46 UTC] PHP Deprecated:  Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Undefined variable $activation_email_address in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2605",
    "level": "WARNING",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:55 UTC] PHP Warning:  Undefined variable $activation_email_address in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2605",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Undefined variable $activation_key in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2610",
    "level": "WARNING",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:55 UTC] PHP Warning:  Undefined variable $activation_key in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2610",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Undefined variable $activation_email_address in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2605",
    "level": "WARNING",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:55 UTC] PHP Warning:  Undefined variable $activation_email_address in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2605",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Undefined variable $activation_key in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2610",
    "level": "WARNING",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:55 UTC] PHP Warning:  Undefined variable $activation_key in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2610",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Undefined variable $activation_email_address in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2605",
    "level": "WARNING",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:03 UTC] PHP Warning:  Undefined variable $activation_email_address in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2605",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Undefined variable $activation_key in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2610",
    "level": "WARNING",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:03 UTC] PHP Warning:  Undefined variable $activation_key in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2610",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Undefined variable $activation_email_address in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2605",
    "level": "WARNING",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:04 UTC] PHP Warning:  Undefined variable $activation_email_address in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2605",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Undefined variable $activation_key in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2610",
    "level": "WARNING",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:04 UTC] PHP Warning:  Undefined variable $activation_key in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2610",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Undefined variable $activation_email_address in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2605",
    "level": "WARNING",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:11 UTC] PHP Warning:  Undefined variable $activation_email_address in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2605",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Undefined variable $activation_key in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2610",
    "level": "WARNING",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:11 UTC] PHP Warning:  Undefined variable $activation_key in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2610",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Undefined variable $activation_email_address in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2605",
    "level": "WARNING",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:12 UTC] PHP Warning:  Undefined variable $activation_email_address in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2605",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Undefined variable $activation_key in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2610",
    "level": "WARNING",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:12 UTC] PHP Warning:  Undefined variable $activation_key in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2610",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Uncaught Error: Call to a member function getName() on null in /wordpress/wp-content/plugins/catalogo/modules/Catalog.php:488",
    "level": "FATAL",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:47 UTC] PHP Fatal error:  Uncaught Error: Call to a member function getName() on null in /wordpress/wp-content/plugins/catalogo/modules/Catalog.php:488\nStack trace:\n#0 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(1171): CatalogoPluginColumnOperations->getIdFromName(Array, 'ref')\n#1 /wordpress/wp-content/plugins/catalogo/catalogo.php(494): CatalogoPluginCatalogData->getTypeColumns(Array, Array)\n#2 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#6 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#7 {main}\n  thrown in /wordpress/wp-content/plugins/catalogo/modules/Catalog.php on line 488",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: _mysql_data_types_cache.mysql_type.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:55 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>CREATE TABLE wp_catalogoplugin_column(\n                id int(11) NOT NULL AUTO_INCREMENT ,\n                name char(32) NOT NULL ,\n                description char(128) ,\n                id_column_definition int NOT NULL,\n                PRIMARY KEY ( `id`  ),\n                FOREIGN KEY (`id_column_definition`) REFERENCES wp_catalogoplugin_column_definition(`id`)\n                ) </p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_catalogoplugin_column, id, int(11)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_catalogoplugin_column, name, char(32)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_catalogoplugin_column, description, char(128)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_catalogoplugin_column, id_column_definition, int</li>\n<li>Executing: CREATE TABLE &quot;wp_catalogoplugin_column&quot; (\n&quot;id&quot; integer PRIMARY KEY AUTOINCREMENT NOT NULL ON CONFLICT REPLACE DEFAULT 0,\n&quot;name&quot; text NOT NULL ON CONFLICT REPLACE DEFAULT '' COLLATE NOCASE,\n&quot;description&quot; text COLLATE NOCASE,\n&quot;id_column_definition&quot; integer NOT NULL ON CONFLICT REPLACE DEFAULT 0) | (no parameters)</li>\n<li>Executing: CREATE  INDEX  &quot;wp_catalogoplugin_column__&quot; ON &quot;wp_catalogoplugin_column&quot; (&quot;&quot;) | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_catalogoplugin_column, wp_catalogoplugin_column__, </li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: _mysql_data_types_cache.mysql_type.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE wp...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(474): dbDelta(Array)\n#3 /wordpress/wp-content/plugins/catalogo/catalogo.php(477): CatalogoPluginColumnOperations->createTable()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#8 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#9 {main}</pre>\n for query CREATE TABLE wp_catalogoplugin_column(\n                id int(11) NOT NULL AUTO_INCREMENT ,\n                name char(32) NOT NULL ,\n                description char(128) ,\n                id_column_definition int NOT NULL,\n                PRIMARY KEY ( `id`  ),\n                FOREIGN KEY (`id_column_definition`) REFERENCES wp_catalogoplugin_column_definition(`id`)\n                )  made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginColumnOperations->createTable, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:55 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>insert into wp_catalogoplugin_column(name, description, id_column_definition) values('ref', 'reference[4] NOT NULL',1)</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: insert into wp_catalogoplugin_column(name, description, id_column_definition) values(:param0 , :param1 ,1) | parameters: ref, reference[4] NOT NULL</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(399): WP_SQLite_DB->query('insert into wp_...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(403): CatalogoPluginColumnOperations->insert(Object(CatalogoPluginColumn))\n#3 /wordpress/wp-content/plugins/catalogo/catalogo.php(481): CatalogoPluginColumnOperations->insertGet(Object(CatalogoPluginColumn))\n#4 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#8 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#9 {main}</pre>\n for query insert into wp_catalogoplugin_column(name, description, id_column_definition) values('ref', 'reference[4] NOT NULL',1) made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginColumnOperations->insertGet, CatalogoPluginColumnOperations->insert, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:55 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select * from wp_catalogoplugin_column where name='ref'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select * from wp_catalogoplugin_column where name=:param0  | parameters: ref</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select * from w...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(448): wpdb->get_results('select * from w...')\n#3 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(404): CatalogoPluginColumnOperations->getByName('ref')\n#4 /wordpress/wp-content/plugins/catalogo/catalogo.php(481): CatalogoPluginColumnOperations->insertGet(Object(CatalogoPluginColumn))\n#5 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#8 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#9 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#10 {main}</pre>\n for query select * from wp_catalogoplugin_column where name='ref' made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginColumnOperations->insertGet, CatalogoPluginColumnOperations->getByName, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:55 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>insert into wp_catalogoplugin_column(name, description, id_column_definition) values('label', 'label[50]',2)</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: insert into wp_catalogoplugin_column(name, description, id_column_definition) values(:param0 , :param1 ,2) | parameters: label, label[50]</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(399): WP_SQLite_DB->query('insert into wp_...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(403): CatalogoPluginColumnOperations->insert(Object(CatalogoPluginColumn))\n#3 /wordpress/wp-content/plugins/catalogo/catalogo.php(481): CatalogoPluginColumnOperations->insertGet(Object(CatalogoPluginColumn))\n#4 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#8 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#9 {main}</pre>\n for query insert into wp_catalogoplugin_column(name, description, id_column_definition) values('label', 'label[50]',2) made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginColumnOperations->insertGet, CatalogoPluginColumnOperations->insert, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:55 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select * from wp_catalogoplugin_column where name='label'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select * from wp_catalogoplugin_column where name=:param0  | parameters: label</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select * from w...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(448): wpdb->get_results('select * from w...')\n#3 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(404): CatalogoPluginColumnOperations->getByName('label')\n#4 /wordpress/wp-content/plugins/catalogo/catalogo.php(481): CatalogoPluginColumnOperations->insertGet(Object(CatalogoPluginColumn))\n#5 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#8 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#9 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#10 {main}</pre>\n for query select * from wp_catalogoplugin_column where name='label' made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginColumnOperations->insertGet, CatalogoPluginColumnOperations->getByName, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:55 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>insert into wp_catalogoplugin_column(name, description, id_column_definition) values('autor', 'autor[50]',3)</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: insert into wp_catalogoplugin_column(name, description, id_column_definition) values(:param0 , :param1 ,3) | parameters: autor, autor[50]</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(399): WP_SQLite_DB->query('insert into wp_...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(403): CatalogoPluginColumnOperations->insert(Object(CatalogoPluginColumn))\n#3 /wordpress/wp-content/plugins/catalogo/catalogo.php(481): CatalogoPluginColumnOperations->insertGet(Object(CatalogoPluginColumn))\n#4 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#8 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#9 {main}</pre>\n for query insert into wp_catalogoplugin_column(name, description, id_column_definition) values('autor', 'autor[50]',3) made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginColumnOperations->insertGet, CatalogoPluginColumnOperations->insert, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:55 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select * from wp_catalogoplugin_column where name='autor'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select * from wp_catalogoplugin_column where name=:param0  | parameters: autor</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select * from w...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(448): wpdb->get_results('select * from w...')\n#3 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(404): CatalogoPluginColumnOperations->getByName('autor')\n#4 /wordpress/wp-content/plugins/catalogo/catalogo.php(481): CatalogoPluginColumnOperations->insertGet(Object(CatalogoPluginColumn))\n#5 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#8 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#9 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#10 {main}</pre>\n for query select * from wp_catalogoplugin_column where name='autor' made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginColumnOperations->insertGet, CatalogoPluginColumnOperations->getByName, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:55 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>insert into wp_catalogoplugin_column(name, description, id_column_definition) values('title', 'title[50]',4)</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: insert into wp_catalogoplugin_column(name, description, id_column_definition) values(:param0 , :param1 ,4) | parameters: title, title[50]</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(399): WP_SQLite_DB->query('insert into wp_...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(403): CatalogoPluginColumnOperations->insert(Object(CatalogoPluginColumn))\n#3 /wordpress/wp-content/plugins/catalogo/catalogo.php(481): CatalogoPluginColumnOperations->insertGet(Object(CatalogoPluginColumn))\n#4 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#8 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#9 {main}</pre>\n for query insert into wp_catalogoplugin_column(name, description, id_column_definition) values('title', 'title[50]',4) made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginColumnOperations->insertGet, CatalogoPluginColumnOperations->insert, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:55 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select * from wp_catalogoplugin_column where name='title'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select * from wp_catalogoplugin_column where name=:param0  | parameters: title</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select * from w...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(448): wpdb->get_results('select * from w...')\n#3 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(404): CatalogoPluginColumnOperations->getByName('title')\n#4 /wordpress/wp-content/plugins/catalogo/catalogo.php(481): CatalogoPluginColumnOperations->insertGet(Object(CatalogoPluginColumn))\n#5 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#8 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#9 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#10 {main}</pre>\n for query select * from wp_catalogoplugin_column where name='title' made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginColumnOperations->insertGet, CatalogoPluginColumnOperations->getByName, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:55 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>insert into wp_catalogoplugin_column(name, description, id_column_definition) values('comment', 'comment[255]',5)</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: insert into wp_catalogoplugin_column(name, description, id_column_definition) values(:param0 , :param1 ,5) | parameters: comment, comment[255]</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(399): WP_SQLite_DB->query('insert into wp_...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(403): CatalogoPluginColumnOperations->insert(Object(CatalogoPluginColumn))\n#3 /wordpress/wp-content/plugins/catalogo/catalogo.php(481): CatalogoPluginColumnOperations->insertGet(Object(CatalogoPluginColumn))\n#4 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#8 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#9 {main}</pre>\n for query insert into wp_catalogoplugin_column(name, description, id_column_definition) values('comment', 'comment[255]',5) made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginColumnOperations->insertGet, CatalogoPluginColumnOperations->insert, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:55 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select * from wp_catalogoplugin_column where name='comment'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select * from wp_catalogoplugin_column where name=:param0  | parameters: comment</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select * from w...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(448): wpdb->get_results('select * from w...')\n#3 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(404): CatalogoPluginColumnOperations->getByName('comment')\n#4 /wordpress/wp-content/plugins/catalogo/catalogo.php(481): CatalogoPluginColumnOperations->insertGet(Object(CatalogoPluginColumn))\n#5 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#8 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#9 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#10 {main}</pre>\n for query select * from wp_catalogoplugin_column where name='comment' made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginColumnOperations->insertGet, CatalogoPluginColumnOperations->getByName, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:55 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>insert into wp_catalogoplugin_column(name, description, id_column_definition) values('storage', 'id of storage[2]',6)</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: insert into wp_catalogoplugin_column(name, description, id_column_definition) values(:param0 , :param1 ,6) | parameters: storage, id of storage[2]</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(399): WP_SQLite_DB->query('insert into wp_...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(403): CatalogoPluginColumnOperations->insert(Object(CatalogoPluginColumn))\n#3 /wordpress/wp-content/plugins/catalogo/catalogo.php(481): CatalogoPluginColumnOperations->insertGet(Object(CatalogoPluginColumn))\n#4 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#8 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#9 {main}</pre>\n for query insert into wp_catalogoplugin_column(name, description, id_column_definition) values('storage', 'id of storage[2]',6) made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginColumnOperations->insertGet, CatalogoPluginColumnOperations->insert, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:55 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select * from wp_catalogoplugin_column where name='storage'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select * from wp_catalogoplugin_column where name=:param0  | parameters: storage</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select * from w...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(448): wpdb->get_results('select * from w...')\n#3 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(404): CatalogoPluginColumnOperations->getByName('storage')\n#4 /wordpress/wp-content/plugins/catalogo/catalogo.php(481): CatalogoPluginColumnOperations->insertGet(Object(CatalogoPluginColumn))\n#5 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#8 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#9 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#10 {main}</pre>\n for query select * from wp_catalogoplugin_column where name='storage' made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginColumnOperations->insertGet, CatalogoPluginColumnOperations->getByName, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: _mysql_data_types_cache.mysql_type.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:55 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>CREATE TABLE wp_catalogoplugin_type_columns(\n                id_type int(11) NOT NULL,\n                id_column int(11) NOT NULL,\n                PRIMARY KEY ( `id_type`, `id_column` ),\n                FOREIGN KEY (`id_type`) REFERENCES wp_catalogoplugin_type(`id`),\n                FOREIGN KEY (`id_column`) REFERENCES wp_catalogoplugin_column(`id`)\n                ) </p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_catalogoplugin_type_columns, id_type, int(11)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_catalogoplugin_type_columns, id_column, int(11)</li>\n<li>Executing: CREATE TABLE &quot;wp_catalogoplugin_type_columns&quot; (\n&quot;id_type&quot; integer NOT NULL ON CONFLICT REPLACE DEFAULT 0,\n&quot;id_column&quot; integer NOT NULL ON CONFLICT REPLACE DEFAULT 0,\nPRIMARY KEY (&quot;id_type&quot;, &quot;id_column&quot;)) | (no parameters)</li>\n<li>Executing: CREATE  INDEX  &quot;wp_catalogoplugin_type_columns__&quot; ON &quot;wp_catalogoplugin_type_columns&quot; (&quot;&quot;) | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_catalogoplugin_type_columns, wp_catalogoplugin_type_columns__, </li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: _mysql_data_types_cache.mysql_type.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE wp...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(638): dbDelta(Array)\n#3 /wordpress/wp-content/plugins/catalogo/catalogo.php(493): CatalogoPluginTypeColumnsOperations->createTable()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#8 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#9 {main}</pre>\n for query CREATE TABLE wp_catalogoplugin_type_columns(\n                id_type int(11) NOT NULL,\n                id_column int(11) NOT NULL,\n                PRIMARY KEY ( `id_type`, `id_column` ),\n                FOREIGN KEY (`id_type`) REFERENCES wp_catalogoplugin_type(`id`),\n                FOREIGN KEY (`id_column`) REFERENCES wp_catalogoplugin_column(`id`)\n                )  made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginTypeColumnsOperations->createTable, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Uncaught Error: Call to a member function getName() on null in /wordpress/wp-content/plugins/catalogo/modules/Catalog.php:488",
    "level": "FATAL",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:55 UTC] PHP Fatal error:  Uncaught Error: Call to a member function getName() on null in /wordpress/wp-content/plugins/catalogo/modules/Catalog.php:488\nStack trace:\n#0 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(1171): CatalogoPluginColumnOperations->getIdFromName(Array, 'ref')\n#1 /wordpress/wp-content/plugins/catalogo/catalogo.php(494): CatalogoPluginCatalogData->getTypeColumns(Array, Array)\n#2 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#6 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#7 {main}\n  thrown in /wordpress/wp-content/plugins/catalogo/modules/Catalog.php on line 488",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: _mysql_data_types_cache.mysql_type.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:55:04 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>CREATE TABLE wp_catalogoplugin_column(\n                id int(11) NOT NULL AUTO_INCREMENT ,\n                name char(32) NOT NULL ,\n                description char(128) ,\n                id_column_definition int NOT NULL,\n                PRIMARY KEY ( `id`  ),\n                FOREIGN KEY (`id_column_definition`) REFERENCES wp_catalogoplugin_column_definition(`id`)\n                ) </p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_catalogoplugin_column, id, int(11)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_catalogoplugin_column, name, char(32)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_catalogoplugin_column, description, char(128)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_catalogoplugin_column, id_column_definition, int</li>\n<li>Executing: CREATE TABLE &quot;wp_catalogoplugin_column&quot; (\n&quot;id&quot; integer PRIMARY KEY AUTOINCREMENT NOT NULL ON CONFLICT REPLACE DEFAULT 0,\n&quot;name&quot; text NOT NULL ON CONFLICT REPLACE DEFAULT '' COLLATE NOCASE,\n&quot;description&quot; text COLLATE NOCASE,\n&quot;id_column_definition&quot; integer NOT NULL ON CONFLICT REPLACE DEFAULT 0) | (no parameters)</li>\n<li>Executing: CREATE  INDEX  &quot;wp_catalogoplugin_column__&quot; ON &quot;wp_catalogoplugin_column&quot; (&quot;&quot;) | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_catalogoplugin_column, wp_catalogoplugin_column__, </li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: _mysql_data_types_cache.mysql_type.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE wp...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(474): dbDelta(Array)\n#3 /wordpress/wp-content/plugins/catalogo/catalogo.php(477): CatalogoPluginColumnOperations->createTable()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#8 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#9 {main}</pre>\n for query CREATE TABLE wp_catalogoplugin_column(\n                id int(11) NOT NULL AUTO_INCREMENT ,\n                name char(32) NOT NULL ,\n                description char(128) ,\n                id_column_definition int NOT NULL,\n                PRIMARY KEY ( `id`  ),\n                FOREIGN KEY (`id_column_definition`) REFERENCES wp_catalogoplugin_column_definition(`id`)\n                )  made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginColumnOperations->createTable, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:55:04 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>insert into wp_catalogoplugin_column(name, description, id_column_definition) values('ref', 'reference[4] NOT NULL',1)</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: insert into wp_catalogoplugin_column(name, description, id_column_definition) values(:param0 , :param1 ,1) | parameters: ref, reference[4] NOT NULL</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(399): WP_SQLite_DB->query('insert into wp_...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(403): CatalogoPluginColumnOperations->insert(Object(CatalogoPluginColumn))\n#3 /wordpress/wp-content/plugins/catalogo/catalogo.php(481): CatalogoPluginColumnOperations->insertGet(Object(CatalogoPluginColumn))\n#4 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#8 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#9 {main}</pre>\n for query insert into wp_catalogoplugin_column(name, description, id_column_definition) values('ref', 'reference[4] NOT NULL',1) made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginColumnOperations->insertGet, CatalogoPluginColumnOperations->insert, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:55:04 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select * from wp_catalogoplugin_column where name='ref'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select * from wp_catalogoplugin_column where name=:param0  | parameters: ref</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select * from w...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(448): wpdb->get_results('select * from w...')\n#3 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(404): CatalogoPluginColumnOperations->getByName('ref')\n#4 /wordpress/wp-content/plugins/catalogo/catalogo.php(481): CatalogoPluginColumnOperations->insertGet(Object(CatalogoPluginColumn))\n#5 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#8 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#9 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#10 {main}</pre>\n for query select * from wp_catalogoplugin_column where name='ref' made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginColumnOperations->insertGet, CatalogoPluginColumnOperations->getByName, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:55:04 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>insert into wp_catalogoplugin_column(name, description, id_column_definition) values('label', 'label[50]',2)</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: insert into wp_catalogoplugin_column(name, description, id_column_definition) values(:param0 , :param1 ,2) | parameters: label, label[50]</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(399): WP_SQLite_DB->query('insert into wp_...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(403): CatalogoPluginColumnOperations->insert(Object(CatalogoPluginColumn))\n#3 /wordpress/wp-content/plugins/catalogo/catalogo.php(481): CatalogoPluginColumnOperations->insertGet(Object(CatalogoPluginColumn))\n#4 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#8 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#9 {main}</pre>\n for query insert into wp_catalogoplugin_column(name, description, id_column_definition) values('label', 'label[50]',2) made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginColumnOperations->insertGet, CatalogoPluginColumnOperations->insert, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:55:04 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select * from wp_catalogoplugin_column where name='label'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select * from wp_catalogoplugin_column where name=:param0  | parameters: label</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select * from w...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(448): wpdb->get_results('select * from w...')\n#3 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(404): CatalogoPluginColumnOperations->getByName('label')\n#4 /wordpress/wp-content/plugins/catalogo/catalogo.php(481): CatalogoPluginColumnOperations->insertGet(Object(CatalogoPluginColumn))\n#5 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#8 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#9 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#10 {main}</pre>\n for query select * from wp_catalogoplugin_column where name='label' made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginColumnOperations->insertGet, CatalogoPluginColumnOperations->getByName, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:55:04 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>insert into wp_catalogoplugin_column(name, description, id_column_definition) values('autor', 'autor[50]',3)</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: insert into wp_catalogoplugin_column(name, description, id_column_definition) values(:param0 , :param1 ,3) | parameters: autor, autor[50]</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(399): WP_SQLite_DB->query('insert into wp_...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(403): CatalogoPluginColumnOperations->insert(Object(CatalogoPluginColumn))\n#3 /wordpress/wp-content/plugins/catalogo/catalogo.php(481): CatalogoPluginColumnOperations->insertGet(Object(CatalogoPluginColumn))\n#4 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#8 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#9 {main}</pre>\n for query insert into wp_catalogoplugin_column(name, description, id_column_definition) values('autor', 'autor[50]',3) made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginColumnOperations->insertGet, CatalogoPluginColumnOperations->insert, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:55:04 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select * from wp_catalogoplugin_column where name='autor'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select * from wp_catalogoplugin_column where name=:param0  | parameters: autor</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select * from w...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(448): wpdb->get_results('select * from w...')\n#3 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(404): CatalogoPluginColumnOperations->getByName('autor')\n#4 /wordpress/wp-content/plugins/catalogo/catalogo.php(481): CatalogoPluginColumnOperations->insertGet(Object(CatalogoPluginColumn))\n#5 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#8 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#9 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#10 {main}</pre>\n for query select * from wp_catalogoplugin_column where name='autor' made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginColumnOperations->insertGet, CatalogoPluginColumnOperations->getByName, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:55:04 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>insert into wp_catalogoplugin_column(name, description, id_column_definition) values('title', 'title[50]',4)</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: insert into wp_catalogoplugin_column(name, description, id_column_definition) values(:param0 , :param1 ,4) | parameters: title, title[50]</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(399): WP_SQLite_DB->query('insert into wp_...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(403): CatalogoPluginColumnOperations->insert(Object(CatalogoPluginColumn))\n#3 /wordpress/wp-content/plugins/catalogo/catalogo.php(481): CatalogoPluginColumnOperations->insertGet(Object(CatalogoPluginColumn))\n#4 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#8 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#9 {main}</pre>\n for query insert into wp_catalogoplugin_column(name, description, id_column_definition) values('title', 'title[50]',4) made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginColumnOperations->insertGet, CatalogoPluginColumnOperations->insert, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:55:04 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select * from wp_catalogoplugin_column where name='title'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select * from wp_catalogoplugin_column where name=:param0  | parameters: title</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select * from w...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(448): wpdb->get_results('select * from w...')\n#3 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(404): CatalogoPluginColumnOperations->getByName('title')\n#4 /wordpress/wp-content/plugins/catalogo/catalogo.php(481): CatalogoPluginColumnOperations->insertGet(Object(CatalogoPluginColumn))\n#5 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#8 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#9 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#10 {main}</pre>\n for query select * from wp_catalogoplugin_column where name='title' made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginColumnOperations->insertGet, CatalogoPluginColumnOperations->getByName, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:55:04 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>insert into wp_catalogoplugin_column(name, description, id_column_definition) values('comment', 'comment[255]',5)</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: insert into wp_catalogoplugin_column(name, description, id_column_definition) values(:param0 , :param1 ,5) | parameters: comment, comment[255]</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(399): WP_SQLite_DB->query('insert into wp_...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(403): CatalogoPluginColumnOperations->insert(Object(CatalogoPluginColumn))\n#3 /wordpress/wp-content/plugins/catalogo/catalogo.php(481): CatalogoPluginColumnOperations->insertGet(Object(CatalogoPluginColumn))\n#4 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#8 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#9 {main}</pre>\n for query insert into wp_catalogoplugin_column(name, description, id_column_definition) values('comment', 'comment[255]',5) made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginColumnOperations->insertGet, CatalogoPluginColumnOperations->insert, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:55:04 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select * from wp_catalogoplugin_column where name='comment'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select * from wp_catalogoplugin_column where name=:param0  | parameters: comment</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select * from w...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(448): wpdb->get_results('select * from w...')\n#3 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(404): CatalogoPluginColumnOperations->getByName('comment')\n#4 /wordpress/wp-content/plugins/catalogo/catalogo.php(481): CatalogoPluginColumnOperations->insertGet(Object(CatalogoPluginColumn))\n#5 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#8 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#9 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#10 {main}</pre>\n for query select * from wp_catalogoplugin_column where name='comment' made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginColumnOperations->insertGet, CatalogoPluginColumnOperations->getByName, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:55:04 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>insert into wp_catalogoplugin_column(name, description, id_column_definition) values('storage', 'id of storage[2]',6)</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: insert into wp_catalogoplugin_column(name, description, id_column_definition) values(:param0 , :param1 ,6) | parameters: storage, id of storage[2]</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(399): WP_SQLite_DB->query('insert into wp_...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(403): CatalogoPluginColumnOperations->insert(Object(CatalogoPluginColumn))\n#3 /wordpress/wp-content/plugins/catalogo/catalogo.php(481): CatalogoPluginColumnOperations->insertGet(Object(CatalogoPluginColumn))\n#4 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#8 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#9 {main}</pre>\n for query insert into wp_catalogoplugin_column(name, description, id_column_definition) values('storage', 'id of storage[2]',6) made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginColumnOperations->insertGet, CatalogoPluginColumnOperations->insert, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:55:04 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select * from wp_catalogoplugin_column where name='storage'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select * from wp_catalogoplugin_column where name=:param0  | parameters: storage</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select * from w...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(448): wpdb->get_results('select * from w...')\n#3 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(404): CatalogoPluginColumnOperations->getByName('storage')\n#4 /wordpress/wp-content/plugins/catalogo/catalogo.php(481): CatalogoPluginColumnOperations->insertGet(Object(CatalogoPluginColumn))\n#5 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#8 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#9 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#10 {main}</pre>\n for query select * from wp_catalogoplugin_column where name='storage' made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginColumnOperations->insertGet, CatalogoPluginColumnOperations->getByName, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: _mysql_data_types_cache.mysql_type.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:55:04 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>CREATE TABLE wp_catalogoplugin_type_columns(\n                id_type int(11) NOT NULL,\n                id_column int(11) NOT NULL,\n                PRIMARY KEY ( `id_type`, `id_column` ),\n                FOREIGN KEY (`id_type`) REFERENCES wp_catalogoplugin_type(`id`),\n                FOREIGN KEY (`id_column`) REFERENCES wp_catalogoplugin_column(`id`)\n                ) </p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_catalogoplugin_type_columns, id_type, int(11)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_catalogoplugin_type_columns, id_column, int(11)</li>\n<li>Executing: CREATE TABLE &quot;wp_catalogoplugin_type_columns&quot; (\n&quot;id_type&quot; integer NOT NULL ON CONFLICT REPLACE DEFAULT 0,\n&quot;id_column&quot; integer NOT NULL ON CONFLICT REPLACE DEFAULT 0,\nPRIMARY KEY (&quot;id_type&quot;, &quot;id_column&quot;)) | (no parameters)</li>\n<li>Executing: CREATE  INDEX  &quot;wp_catalogoplugin_type_columns__&quot; ON &quot;wp_catalogoplugin_type_columns&quot; (&quot;&quot;) | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_catalogoplugin_type_columns, wp_catalogoplugin_type_columns__, </li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: _mysql_data_types_cache.mysql_type.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE wp...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(638): dbDelta(Array)\n#3 /wordpress/wp-content/plugins/catalogo/catalogo.php(493): CatalogoPluginTypeColumnsOperations->createTable()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#8 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#9 {main}</pre>\n for query CREATE TABLE wp_catalogoplugin_type_columns(\n                id_type int(11) NOT NULL,\n                id_column int(11) NOT NULL,\n                PRIMARY KEY ( `id_type`, `id_column` ),\n                FOREIGN KEY (`id_type`) REFERENCES wp_catalogoplugin_type(`id`),\n                FOREIGN KEY (`id_column`) REFERENCES wp_catalogoplugin_column(`id`)\n                )  made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginTypeColumnsOperations->createTable, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Uncaught Error: Call to a member function getName() on null in /wordpress/wp-content/plugins/catalogo/modules/Catalog.php:488",
    "level": "FATAL",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:55:04 UTC] PHP Fatal error:  Uncaught Error: Call to a member function getName() on null in /wordpress/wp-content/plugins/catalogo/modules/Catalog.php:488\nStack trace:\n#0 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(1171): CatalogoPluginColumnOperations->getIdFromName(Array, 'ref')\n#1 /wordpress/wp-content/plugins/catalogo/catalogo.php(494): CatalogoPluginCatalogData->getTypeColumns(Array, Array)\n#2 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#6 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#7 {main}\n  thrown in /wordpress/wp-content/plugins/catalogo/modules/Catalog.php on line 488",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Uncaught ArgumentCountError: Too few arguments to function WP_Widget::__construct(), 0 passed in /wordpress/wp-includes/class-wp-widget-factory.php on line 62 and at least 2 expected in /wordpress/wp-includes/class-wp-widget.php:163",
    "level": "FATAL",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:55:45 UTC] PHP Fatal error:  Uncaught ArgumentCountError: Too few arguments to function WP_Widget::__construct(), 0 passed in /wordpress/wp-includes/class-wp-widget-factory.php on line 62 and at least 2 expected in /wordpress/wp-includes/class-wp-widget.php:163\nStack trace:\n#0 /wordpress/wp-includes/class-wp-widget-factory.php(62): WP_Widget->__construct()\n#1 /wordpress/wp-includes/widgets.php(123): WP_Widget_Factory->register('widget_cbnet_tw...')\n#2 /wordpress/wp-content/plugins/cbnet-twitter-widget/cbnet-twitter-widget.php(47): register_widget('widget_cbnet_tw...')\n#3 /wordpress/wp-includes/class-wp-hook.php(324): cbnet_twitter_widget_load_widget('')\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-includes/widgets.php(1870): do_action('widgets_init')\n#7 /wordpress/wp-includes/class-wp-hook.php(324): wp_widgets_init('')\n#8 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#9 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#10 /wordpress/wp-settings.php(727): do_action('init')\n#11 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#12 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#13 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#14 {main}\n  thrown in /wordpress/wp-includes/class-wp-widget.php on line 163",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Uncaught ArgumentCountError: Too few arguments to function WP_Widget::__construct(), 0 passed in /wordpress/wp-includes/class-wp-widget-factory.php on line 62 and at least 2 expected in /wordpress/wp-includes/class-wp-widget.php:163",
    "level": "FATAL",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:55:53 UTC] PHP Fatal error:  Uncaught ArgumentCountError: Too few arguments to function WP_Widget::__construct(), 0 passed in /wordpress/wp-includes/class-wp-widget-factory.php on line 62 and at least 2 expected in /wordpress/wp-includes/class-wp-widget.php:163\nStack trace:\n#0 /wordpress/wp-includes/class-wp-widget-factory.php(62): WP_Widget->__construct()\n#1 /wordpress/wp-includes/widgets.php(123): WP_Widget_Factory->register('widget_cbnet_tw...')\n#2 /wordpress/wp-content/plugins/cbnet-twitter-widget/cbnet-twitter-widget.php(47): register_widget('widget_cbnet_tw...')\n#3 /wordpress/wp-includes/class-wp-hook.php(324): cbnet_twitter_widget_load_widget('')\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-includes/widgets.php(1870): do_action('widgets_init')\n#7 /wordpress/wp-includes/class-wp-hook.php(324): wp_widgets_init('')\n#8 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#9 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#10 /wordpress/wp-settings.php(727): do_action('init')\n#11 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#12 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#13 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#14 {main}\n  thrown in /wordpress/wp-includes/class-wp-widget.php on line 163",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Uncaught ArgumentCountError: Too few arguments to function WP_Widget::__construct(), 0 passed in /wordpress/wp-includes/class-wp-widget-factory.php on line 62 and at least 2 expected in /wordpress/wp-includes/class-wp-widget.php:163",
    "level": "FATAL",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:56:01 UTC] PHP Fatal error:  Uncaught ArgumentCountError: Too few arguments to function WP_Widget::__construct(), 0 passed in /wordpress/wp-includes/class-wp-widget-factory.php on line 62 and at least 2 expected in /wordpress/wp-includes/class-wp-widget.php:163\nStack trace:\n#0 /wordpress/wp-includes/class-wp-widget-factory.php(62): WP_Widget->__construct()\n#1 /wordpress/wp-includes/widgets.php(123): WP_Widget_Factory->register('widget_cbnet_tw...')\n#2 /wordpress/wp-content/plugins/cbnet-twitter-widget/cbnet-twitter-widget.php(47): register_widget('widget_cbnet_tw...')\n#3 /wordpress/wp-includes/class-wp-hook.php(324): cbnet_twitter_widget_load_widget('')\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-includes/widgets.php(1870): do_action('widgets_init')\n#7 /wordpress/wp-includes/class-wp-hook.php(324): wp_widgets_init('')\n#8 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#9 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#10 /wordpress/wp-settings.php(727): do_action('init')\n#11 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#12 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#13 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#14 {main}\n  thrown in /wordpress/wp-includes/class-wp-widget.php on line 163",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Uncaught Error: Call to undefined method CF7_BETTER_UI::getSettings() in /wordpress/wp-content/plugins/cf7-better-ui/class-cf7-better-ui.php:29",
    "level": "FATAL",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:56:10 UTC] PHP Fatal error:  Uncaught Error: Call to undefined method CF7_BETTER_UI::getSettings() in /wordpress/wp-content/plugins/cf7-better-ui/class-cf7-better-ui.php:29\nStack trace:\n#0 /wordpress/wp-content/plugins/cf7-better-ui/class-cf7-better-ui.php(195): CF7_BETTER_UI->__construct()\n#1 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#2 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('cf7-better-ui/c...')\n#3 /internal/eval.php(20): activate_plugin('cf7-better-ui/c...')\n#4 {main}\n  thrown in /wordpress/wp-content/plugins/cf7-better-ui/class-cf7-better-ui.php on line 29",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Uncaught Error: Call to undefined method CF7_BETTER_UI::getSettings() in /wordpress/wp-content/plugins/cf7-better-ui/class-cf7-better-ui.php:29",
    "level": "FATAL",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:56:17 UTC] PHP Fatal error:  Uncaught Error: Call to undefined method CF7_BETTER_UI::getSettings() in /wordpress/wp-content/plugins/cf7-better-ui/class-cf7-better-ui.php:29\nStack trace:\n#0 /wordpress/wp-content/plugins/cf7-better-ui/class-cf7-better-ui.php(195): CF7_BETTER_UI->__construct()\n#1 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#2 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('cf7-better-ui/c...')\n#3 /internal/eval.php(20): activate_plugin('cf7-better-ui/c...')\n#4 {main}\n  thrown in /wordpress/wp-content/plugins/cf7-better-ui/class-cf7-better-ui.php on line 29\n\n    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n  [cause]: Error: Error when executing the blueprint step #1 ({\"step\":\"installPlugin\",\"pluginData\":{\"resource\":\"wordpress.org/plugins\",\"slug\":\"cf7-better-ui\"},\"options\":{\"activate\":true}}) : PHP.run() failed with exit code 255 and the following output: \n      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n      ... 2 lines matching cause stack trace ...\n      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n    [cause]: PHPExecutionFailureError: PHP.run() failed with exit code 255 and the following output: \n        at PHP.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/universal/index.js:987:19)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async wr (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:33:39)\n        at async Object.gt [as installPlugin] (file:///home/ru\n",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "level": "FATAL",
    "type": "PLAYGROUND",
    "test": "jspi-boot",
    "plugin": "cf7-better-ui",
    "details": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "log": "/tmp/tmp.h1D0zfyXxP"
  },
  {
    "message": "Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121",
    "level": "DEPRECATED",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:23 UTC] PHP Deprecated:  Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/calicotek-ebay-dashboard-tools/index.php on line 54",
    "level": "WARNING",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:23 UTC] PHP Warning:  Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/calicotek-ebay-dashboard-tools/index.php on line 54",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121",
    "level": "DEPRECATED",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:24 UTC] PHP Deprecated:  Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121",
    "level": "DEPRECATED",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:35 UTC] PHP Deprecated:  Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/calicotek-ebay-dashboard-tools/index.php on line 54",
    "level": "WARNING",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:35 UTC] PHP Warning:  Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/calicotek-ebay-dashboard-tools/index.php on line 54",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121",
    "level": "DEPRECATED",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:36 UTC] PHP Deprecated:  Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121",
    "level": "DEPRECATED",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:45 UTC] PHP Deprecated:  Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/calicotek-ebay-dashboard-tools/index.php on line 54",
    "level": "WARNING",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:45 UTC] PHP Warning:  Trying to access array offset on value of type bool in /wordpress/wp-content/plugins/calicotek-ebay-dashboard-tools/index.php on line 54",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121",
    "level": "DEPRECATED",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:46 UTC] PHP Deprecated:  Function load_plugin_textdomain was called with an argument that is <strong>deprecated</strong> since version 2.7.0 with no alternative available. in /wordpress/wp-includes/functions.php on line 6121",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Undefined variable $activation_email_address in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2605",
    "level": "WARNING",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:55 UTC] PHP Warning:  Undefined variable $activation_email_address in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2605",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Undefined variable $activation_key in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2610",
    "level": "WARNING",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:55 UTC] PHP Warning:  Undefined variable $activation_key in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2610",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Undefined variable $activation_email_address in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2605",
    "level": "WARNING",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:55 UTC] PHP Warning:  Undefined variable $activation_email_address in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2605",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Undefined variable $activation_key in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2610",
    "level": "WARNING",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:53:55 UTC] PHP Warning:  Undefined variable $activation_key in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2610",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Undefined variable $activation_email_address in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2605",
    "level": "WARNING",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:03 UTC] PHP Warning:  Undefined variable $activation_email_address in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2605",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Undefined variable $activation_key in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2610",
    "level": "WARNING",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:03 UTC] PHP Warning:  Undefined variable $activation_key in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2610",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Undefined variable $activation_email_address in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2605",
    "level": "WARNING",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:04 UTC] PHP Warning:  Undefined variable $activation_email_address in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2605",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Undefined variable $activation_key in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2610",
    "level": "WARNING",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:04 UTC] PHP Warning:  Undefined variable $activation_key in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2610",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Undefined variable $activation_email_address in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2605",
    "level": "WARNING",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:11 UTC] PHP Warning:  Undefined variable $activation_email_address in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2605",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Undefined variable $activation_key in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2610",
    "level": "WARNING",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:11 UTC] PHP Warning:  Undefined variable $activation_key in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2610",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Undefined variable $activation_email_address in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2605",
    "level": "WARNING",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:12 UTC] PHP Warning:  Undefined variable $activation_email_address in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2605",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Undefined variable $activation_key in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2610",
    "level": "WARNING",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:12 UTC] PHP Warning:  Undefined variable $activation_key in /wordpress/wp-content/plugins/candidate-application-form/apply_form.php on line 2610",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "Uncaught Error: Call to a member function getName() on null in /wordpress/wp-content/plugins/catalogo/modules/Catalog.php:488",
    "level": "FATAL",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:47 UTC] PHP Fatal error:  Uncaught Error: Call to a member function getName() on null in /wordpress/wp-content/plugins/catalogo/modules/Catalog.php:488\nStack trace:\n#0 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(1171): CatalogoPluginColumnOperations->getIdFromName(Array, 'ref')\n#1 /wordpress/wp-content/plugins/catalogo/catalogo.php(494): CatalogoPluginCatalogData->getTypeColumns(Array, Array)\n#2 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#6 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#7 {main}\n  thrown in /wordpress/wp-content/plugins/catalogo/modules/Catalog.php on line 488",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: _mysql_data_types_cache.mysql_type.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:55 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>CREATE TABLE wp_catalogoplugin_column(\n                id int(11) NOT NULL AUTO_INCREMENT ,\n                name char(32) NOT NULL ,\n                description char(128) ,\n                id_column_definition int NOT NULL,\n                PRIMARY KEY ( `id`  ),\n                FOREIGN KEY (`id_column_definition`) REFERENCES wp_catalogoplugin_column_definition(`id`)\n                ) </p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_catalogoplugin_column, id, int(11)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_catalogoplugin_column, name, char(32)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_catalogoplugin_column, description, char(128)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_catalogoplugin_column, id_column_definition, int</li>\n<li>Executing: CREATE TABLE &quot;wp_catalogoplugin_column&quot; (\n&quot;id&quot; integer PRIMARY KEY AUTOINCREMENT NOT NULL ON CONFLICT REPLACE DEFAULT 0,\n&quot;name&quot; text NOT NULL ON CONFLICT REPLACE DEFAULT '' COLLATE NOCASE,\n&quot;description&quot; text COLLATE NOCASE,\n&quot;id_column_definition&quot; integer NOT NULL ON CONFLICT REPLACE DEFAULT 0) | (no parameters)</li>\n<li>Executing: CREATE  INDEX  &quot;wp_catalogoplugin_column__&quot; ON &quot;wp_catalogoplugin_column&quot; (&quot;&quot;) | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_catalogoplugin_column, wp_catalogoplugin_column__, </li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: _mysql_data_types_cache.mysql_type.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE wp...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(474): dbDelta(Array)\n#3 /wordpress/wp-content/plugins/catalogo/catalogo.php(477): CatalogoPluginColumnOperations->createTable()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#8 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#9 {main}</pre>\n for query CREATE TABLE wp_catalogoplugin_column(\n                id int(11) NOT NULL AUTO_INCREMENT ,\n                name char(32) NOT NULL ,\n                description char(128) ,\n                id_column_definition int NOT NULL,\n                PRIMARY KEY ( `id`  ),\n                FOREIGN KEY (`id_column_definition`) REFERENCES wp_catalogoplugin_column_definition(`id`)\n                )  made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginColumnOperations->createTable, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:55 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>insert into wp_catalogoplugin_column(name, description, id_column_definition) values('ref', 'reference[4] NOT NULL',1)</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: insert into wp_catalogoplugin_column(name, description, id_column_definition) values(:param0 , :param1 ,1) | parameters: ref, reference[4] NOT NULL</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(399): WP_SQLite_DB->query('insert into wp_...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(403): CatalogoPluginColumnOperations->insert(Object(CatalogoPluginColumn))\n#3 /wordpress/wp-content/plugins/catalogo/catalogo.php(481): CatalogoPluginColumnOperations->insertGet(Object(CatalogoPluginColumn))\n#4 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#8 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#9 {main}</pre>\n for query insert into wp_catalogoplugin_column(name, description, id_column_definition) values('ref', 'reference[4] NOT NULL',1) made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginColumnOperations->insertGet, CatalogoPluginColumnOperations->insert, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:55 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select * from wp_catalogoplugin_column where name='ref'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select * from wp_catalogoplugin_column where name=:param0  | parameters: ref</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select * from w...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(448): wpdb->get_results('select * from w...')\n#3 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(404): CatalogoPluginColumnOperations->getByName('ref')\n#4 /wordpress/wp-content/plugins/catalogo/catalogo.php(481): CatalogoPluginColumnOperations->insertGet(Object(CatalogoPluginColumn))\n#5 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#8 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#9 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#10 {main}</pre>\n for query select * from wp_catalogoplugin_column where name='ref' made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginColumnOperations->insertGet, CatalogoPluginColumnOperations->getByName, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:55 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>insert into wp_catalogoplugin_column(name, description, id_column_definition) values('label', 'label[50]',2)</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: insert into wp_catalogoplugin_column(name, description, id_column_definition) values(:param0 , :param1 ,2) | parameters: label, label[50]</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(399): WP_SQLite_DB->query('insert into wp_...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(403): CatalogoPluginColumnOperations->insert(Object(CatalogoPluginColumn))\n#3 /wordpress/wp-content/plugins/catalogo/catalogo.php(481): CatalogoPluginColumnOperations->insertGet(Object(CatalogoPluginColumn))\n#4 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#8 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#9 {main}</pre>\n for query insert into wp_catalogoplugin_column(name, description, id_column_definition) values('label', 'label[50]',2) made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginColumnOperations->insertGet, CatalogoPluginColumnOperations->insert, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  },
  {
    "message": "SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "cf7-better-ui",
    "details": "[08-May-2025 04:54:55 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>select * from wp_catalogoplugin_column where name='label'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: select * from wp_catalogoplugin_column where name=:param0  | parameters: label</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_catalogoplugin_column.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('select * from w...')\n#2 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(448): wpdb->get_results('select * from w...')\n#3 /wordpress/wp-content/plugins/catalogo/modules/Catalog.php(404): CatalogoPluginColumnOperations->getByName('label')\n#4 /wordpress/wp-content/plugins/catalogo/catalogo.php(481): CatalogoPluginColumnOperations->insertGet(Object(CatalogoPluginColumn))\n#5 /wordpress/wp-includes/class-wp-hook.php(324): catalogoPlugin_activationHook(false)\n#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#8 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_catalo...', false)\n#9 /internal/eval.php(20): activate_plugin('catalogo/catalo...')\n#10 {main}</pre>\n for query select * from wp_catalogoplugin_column where name='label' made by activate_plugin, do_action('activate_catalogo/catalogo.php'), WP_Hook->do_action, WP_Hook->apply_filters, catalogoPlugin_activationHook, CatalogoPluginColumnOperations->insertGet, CatalogoPluginColumnOperations->getByName, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/c/cf7-better-ui/error.json"
  }
]
